src/tokens/selectors.ts

```diff
@@ -366,6 +368,7 @@ export const swappableToTokensByNetworkIdSelector = createSelector(
     return tokens.filter(
       (tokenInfo) =>
         tokenInfo.isSwappable ||
+        tokenInfo.isManuallyImported ||
```

Comment:
do you need this when the condition is already there in `swappableFromTokensByNetworkIdSelector`?

Reply:
Hmm, I guess we have to, right?
Because if not, those filters will filter it out?

That being said, I don't like that the `toToken` are built from the `fromToken`. I imagine a case where we want a token to be in the `toToken` list but not in the `fromToken` (for example, tokens without balance).
I think both selector should use the whole token list.

Reply:
yes you're right, my brain confused the intention of the .filter here yesterday.

and i think i understand the concern with the shared selectors - i originally built these selectors totally separate and updated these to share code to make it easier to see the difference between the 2. i agree in the future if we find the selectors are diverging then we should totally separate them completely